docs/nomenclature.md

```diff
@@ -0,0 +1,150 @@
+# Evolution Platform Nomenclature
+
+## Process Names
+
+### Interview Process
+* **Interview**
+   Base process for all interview types
+
+   **InterviewResponse** (shorthand: **Response**)
+   Includes all the responses to questions in the interview from a respondent or an interviewer (for a specific interview)
+
+   **InterviewAnswer** (shorthand: **Answer**)
+   Response/answer to a specific question, field or attribute during the interview
+
+   a. **RespondentInterview**
+      - Process where a respondent completes their own interview
+      - Can be incomplete/abandoned
+      - Creates a Response
+   
+   b. **AssistedInterview**
+      - Interview conducted by an interviewer (phone/chat/in-person)
+      - Can be starting fresh or continuing a RespondentInterview
+      - Even if a RespondentInterview has been started (creating a Response), as soon as an interviewer touches the interview, it becomes an AssistedInterview with an AssistedResponse via a boolean flag for detection during audit
+      - Creates a Response
+
+   Respondent and assisted responses are identical in the flow process and are both called Response.
+   
+* **RealTimeValidation** (shorthand: **Validation**)
+   - Frontend/backend validation during interview
+   - Data type checking
+   - Answers coherence verification
+   - Applies to both interview types and to correction process (see below)
+
+### Post-Interview Processing
+1. **Audit** Automated backend process analyzing respondent and assisted interviews
+    * **AuditCheck**: a single validation/check. Example: check if household size is consistent with the number of persons objects in the interview response. Each auditCheck as a type: error (which must be corrected), warning (which may be corrected but could be ignored), or note (just a message to classify the interview like "has at least one transit trip", "is a single person interview")
```

Comment:
typo "each auditCheck has* a type".

Aussi "which must be corrected", on ne peut pas nécessairement faire de correction, "which should be reviewed and eventually corrected or rejected"?

Idem pour warning, c'est le genre de flag qui pourrait mener à une invalidation de l'entrevue

Reply:
ajouté